instrumentation/android-log/library/src/main/java/io/opentelemetry/instrumentation/library/log/AndroidLogSubstitutions.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.library.log;
+
+import static io.opentelemetry.instrumentation.library.log.internal.LogRecordBuilderCreator.createLogRecordBuilder;
+import static io.opentelemetry.instrumentation.library.log.internal.LogRecordBuilderCreator.getEventName;
+import static io.opentelemetry.instrumentation.library.log.internal.LogRecordBuilderCreator.printStacktrace;
+
+import android.util.Log;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.logs.Severity;
+
+public class AndroidLogSubstitutions {
+
+    public static String TAG_KEY = "android.tag";
```

Comment:
Actually, the guidance is more to use the AttributeKey class, because avoids having to make a new AttributeKey each time (like the pure string version does).

```suggestion
    public final static AttributeKey<String> TAG_KEY = AttributeKey.stringKey("android.tag");
```